cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_svd.cpp

```diff
@@ -14,17 +14,101 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include <daal/src/algorithms/pca/pca_dense_svd_batch_kernel.h>
+#include <daal/include/algorithms/normalization/zscore_types.h>
+
 #include "oneapi/dal/algo/pca/backend/cpu/train_kernel.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/error_converter.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+#include "oneapi/dal/table/row_accessor.hpp"
 
 namespace oneapi::dal::pca::backend {
 
+using std::int64_t;
+using dal::backend::context_cpu;
+
+namespace daal_pca = daal::algorithms::pca;
+namespace interop = dal::backend::interop;
+
+template <typename Float, daal::CpuType Cpu>
+using daal_pca_svd_kernel_t = daal_pca::internal::
+    PCASVDBatchKernel<Float, daal_pca::interface3::BatchParameter<Float, daal_pca::svdDense>, Cpu>;
+
+template <typename Float, typename Task>
+static train_result<Task> call_daal_kernel(const context_cpu& ctx,
```

Comment:
Suggestion: do not depend on template parameters, which are not used, it's potentially increases instantiation complexity and hence compilation time. What can be done:
1. Introduce usings for input, descriptor and result in the very beginning of cpp:
   ```c++
   using input_t = train_input<task::dim_reduction>;
   ...
   ```
2. Remove tasks from template parameters of `call_daal_kernel`, `train` and introduce specialization for `train_kernel_cpu` (I see, it's already done).
3. See how it's done in GPU version of cov PCA.

Reply:
fixed